cgi-bin/request_activation

```diff
@@ -88,27 +88,29 @@ if (!defined($code)) {
     $code = &CleanCode($code);
     my $expected_code = &ActivationCode($address);
     if ($code eq $expected_code) {
+        my $report;
         if (UserActivated($optouts, $address)) {
-            print 'already ', $tx->already_activated;
+            $report = 'already ' + $tx->already_activated;
         } else {
-            my $pats = SetOptOutPatterns($optouts, $address, '+');
-            SaveOptOuts($optouts);
-            my $key = &OptOutKey(&CanonicalizeAddr($address));
-            my $fname = "@CIVSDATADIR@/invites/$key";
-            my @rows = ();
-            my $invites = "";
-            if (open(INVITES, "<", $fname)) {
-                $invites = <INVITES>;
-                my @lines = split /\n/, $invites;
-                for my $line (@lines) {
-                    chomp $line;
-                    (my $url, my $title) = split / /, $line, 2;
-                    push @rows, [$url, $title];
-                }
-                close(INVITES)
+            $report = 'activated ' + $tx->activation_successful;
+        }
+        my $pats = SetOptOutPatterns($optouts, $address, '+');
+        SaveOptOuts($optouts);
+        my $key = &OptOutKey(&CanonicalizeAddr($address));
+        my $fname = "@CIVSDATADIR@/invites/$key";
+        my @rows = ();
+        my $invites = "";
+        if (open(INVITES, "<", $fname)) {
+            $invites = <INVITES>;
+            my @lines = split /\n/, $invites;
+            for my $line (@lines) {
+                chomp $line;
+                (my $url, my $title) = split / /, $line, 2;
+                push @rows, [$url, $title];
             }
-            print 'activated ', $tx->activation_successful($pats, \@rows);
+            close(INVITES)
         }
+        print $report, $tx->pending_invites($pats, \@rows);
```

Comment:
We want a space after $report: print $report, ' ', ....

Reply:
Not positive, but I think this still needs to be changed to
```
print $report, $tx->pending_invites($pats, \@rows);

Reply:
I've double-checked this now, and it behaves correctly.
![activated-pending](https://user-images.githubusercontent.com/1110841/134578061-a08de5e2-861b-485f-9ea2-00e3c6ed7d55.png)